diff --git a/pythonforandroid/recipes/mobileinsight/__init__.py b/pythonforandroid/recipes/mobileinsight/__init__.py
index ad6e4ab..3c0adb0 100644
--- a/pythonforandroid/recipes/mobileinsight/__init__.py
+++ b/pythonforandroid/recipes/mobileinsight/__init__.py
@@ -7,13 +7,13 @@ from os.path import exists, join, isdir, split
 import sh
 import glob
 
-LOCAL_DEBUG = False
+LOCAL_DEBUG = True
 
 class MobileInsightRecipe(Recipe):
 
     mi_git            = 'https://github.com/xg-wang/mobileinsight-core.git'
     mi_branch         = 'vis-proj'
-    local_src         = '/home/vagrant/mi-dev/mobileinsight-core'
+    local_src         = '/vagrant/mi-dev/mobileinsight-core'
     version           = '2.5'
     toolchain_version = 4.9          # default GCC toolchain version we try to use
     depends           = ['python2']  # any other recipe names that must be built before this one
